assets/src/components/account/AccountOverview.tsx

```diff
@@ -29,6 +30,17 @@ type State = {
   isLoading: boolean;
   isEditing: boolean;
   isRefreshing: boolean;
+  hideWidgetOutsideWorkingHours: boolean;
+};
+type Account = {
+  id: string;
+  company_name: string;
+  time_zone: string;
+  subscription_plan: string;
+  hide_widget_outside_working_hours: boolean;
+  users: Array<User>;
+  widget_settings: Array<WidgetSettings>;
```

Comment:
i think this should just be `widget_settings: WidgetSettings`, sorry for the confusing naming 😅